### DISCUSSION TITLE
Is there support for component testing in cypress 10?

### DISCUSSION THREAD

**Discussion by to-jo** [Shelex/cypress-allure-plugin#169] [Q&A]
I really love using allure to document the app through tests.

we are thinking about moving some of our tests over to component tests, but it would still be nice to see the components tests in allure.
I tried configuring this but it doesn't seem to work, any plans to add in the near future? or any suggestions to try?

**Comment by Shelex** [comment]
Hi @to-jo 
If you mean if it should work for cypress `ct` mode - not sure, to be honest I have never seen allure reporter being used for component tests.
Technically, if cypress use same mocha under the hood, same api, same chainable queue, same events -  it should work just out of the box.

However, I will make a not for myself to check plugin with `component` mode.

**Reply by to-jo** [reply]
@Shelex would be great if you can check thanks!

we currently use the jest allure plugin for our component tests, but would like to move over to cypress but still have the documentation of the components' behaviours in the allure report. The e2e part would then only be needed for testing everything works together not individual component behaviour.
